src/features/surveys/components/SubmissionWarningCard.tsx

```diff
@@ -0,0 +1,68 @@
+import messageIds from '../l10n/messageIds';
+import SurveyDataModel from '../models/SurveyDataModel';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import ZUIFuture from 'zui/ZUIFuture';
+import { Alert, AlertTitle, Box, Link } from '@mui/material';
+
+type SubmissionWarningCardProps = {
+  campId: number;
+  filterQuery: string | boolean;
+  orgId: number;
+  surveyId: number;
+};
+const SubmissionWarningCard = ({
+  campId,
+  filterQuery,
+  orgId,
+  surveyId,
+}: SubmissionWarningCardProps) => {
+  const model = useModel((env) => new SurveyDataModel(env, orgId, surveyId));
+  const messages = useMessages(messageIds);
+
+  return (
+    <ZUIFuture future={model.getStats()}>
+      {(sub) => {
+        const unlinkedSubs = sub.unlinkedSubmissionCount;
+
+        return (
+          <>
+            {unlinkedSubs > 0 && (
+              <Alert severity="warning" sx={{ marginLeft: 2 }}>
```

Comment:
There should be no margin defined here. Defining an _external_ margin _inside_ a component should always be a red flag. What if you want to use this component somewhere where this margin does not make sense?

Luckily, this margin can be solved in another way (see separate comment.)